examples/tutorials/rigid_object_tutorial.py

```diff
@@ -148,23 +207,37 @@ def simulate(sim, dt=1.0, get_frames=True):
 
 
 # [/setup]
-
-# This is wrapped such that it can be added to a unit test
-def main(make_video=True, show_video=True):
```

Comment:
The current testing methodology ([tests/test_examples.py](https://github.com/facebookresearch/habitat-sim/blob/f2168529fff262f6d84dfaed1497b8c407e5c1f6/tests/test_examples.py#L35))  is built around around explicitly calling main, so if we are going to change this we need to modify that as well, and modify it in all the other scripts that are tested by test_examples.py.

Reply:
@jturner65 Not anymore, just finished rewriting it.